docs/dashboards/cortex-read.json

```diff
@@ -0,0 +1,894 @@
+{
+   "annotations": {
+      "list": [ ]
+   },
+   "editable": true,
+   "gnetId": null,
+   "graphTooltip": 0,
+   "hideControls": false,
+   "id": null,
+   "links": [ ],
+   "panels": [
+      {
+         "collapsed": false,
+         "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 0
+         },
+         "id": 2,
+         "panels": [ ],
+         "repeat": null,
+         "title": "Querier",
+         "type": "row"
+      },
+      {
+         "aliasColors": {
+            "200": "#7EB26D",
+            "5xx": "#BF1B00"
+         },
+         "bars": false,
+         "dashLength": 10,
+         "dashes": false,
+         "datasource": null,
+         "fill": 10,
+         "gridPos": {
+            "h": 7,
+            "w": 12,
+            "x": 0,
+            "y": 1
+         },
+         "id": 3,
+         "legend": {
+            "alignAsTable": false,
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "rightSide": false,
+            "show": true,
+            "total": false,
+            "values": false
+         },
+         "lines": true,
+         "linewidth": 1,
+         "nullPointMode": "null as zero",
+         "percentage": false,
+         "pointradius": 5,
+         "points": false,
+         "renderer": "flot",
+         "repeat": null,
+         "seriesOverrides": [ ],
+         "spaceLength": 10,
+         "span": 12,
+         "stack": true,
+         "steppedLine": false,
+         "targets": [
+            {
+               "datasource": "default",
+               "expr": "sum(rate(cortex_request_duration_seconds_count{name=\"querier\", status_code=~\"2.*\", route=\"api_prom_api_v1\"}[1m]))",
```

Comment:
Hmm... This does come from our Kubernetes label map scheme. We'll try to figure out a more generic way to determine these. Just using the route or method, though somewhat unique, would return slightly less useful data because it would conflate components that use other components.